packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/Timeline/BasicTimeline/AxisWithNoTickValues.2.doc.mdx

```diff
@@ -11,21 +10,17 @@ This is a basic timeline graph with no axis ticks.
 ## Getting Started
 
 - Install with [npmjs](https://www.npmjs.com):
-  - `npm install terra-graphs`
+  - `npm i @cerner/carbon-graphs --save-dev`
```

Comment:
Should we be suggesting consumers install this package as a devDependency?

Reply:
yes Tyler, that is how we are suggesting consumers even before we merged into terra.